sdk/schemaregistry/azure-data-schemaregistry/src/main/java/com/azure/data/schemaregistry/SchemaRegistryClientBuilder.java

```diff
@@ -77,7 +76,7 @@ public CachedSchemaRegistryClientBuilder() {
         this.retryPolicy = new RetryPolicy("retry-after-ms", ChronoUnit.MILLIS);
 
         HttpHeaders headers = new HttpHeaders();
-        headers.put("api-version", "2017-04");
+        headers.put("api-version", "2020-09-01-preview");
```

Comment:
Looks like the service caused a regression.  It went from allowing api-version as header to blocking it.  We are fixing it on the service side to accept header again.